### DISCUSSION TITLE
Anysoftkeyboard UK currency issue

### DISCUSSION THREAD

**Discussion by Cool4catz** [community/community#29335] [Other Feature Feedback, Questions, & Ideas] [answered]
I have downloaded and installed AnySoftKeyboard for the English language on my phone, but there isn't a UK currency 'pound'' symbol.
Please could someone tell me if there is a version of AnySoftKeyboard including the symbol?
Thanks.

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/AnySoftKeyboard/AnySoftKeyboard? In that case you should open an issue or discussion *in that repository* so the developers see it. :slightly_smiling_face:

**Reply by Cool4catz** [reply]
Ah right, thank you so much 🙂
